terraform/main.tf

```diff
@@ -0,0 +1,115 @@
+terraform {
+  required_version = "~> 1.3"
+
+  required_providers {
+    digitalocean = {
+      source  = "digitalocean/digitalocean"
+      version = "~> 2.0"
+    }
+  }
+}
+
+provider "digitalocean" {
+  token = var.digitalocean_token
+}
+
+# resource "digitalocean_ssh_key" "new-key" {
+#   name       = "new-key"
+#   public_key = file("/Users/mac/.ssh/id_rsa.pub")
+# }
+
+# resource "digitalocean_spaces_bucket" "spaces-name" {
+#   name   = "spaces-name"
+#   region = "nyc3"
+# }
+
+resource "digitalocean_volume" "forest-volum" {
+  region                  = "nyc3"
+  name                    = "forest-volum"
+  size                    = 600
+  initial_filesystem_type = "ext4"
+  description             = "forest storage volume"
+}
+
+resource "digitalocean_droplet" "forest-samuel" {
+  image  = var.image
+  name   = var.name
+  region = var.region
+  size   = var.size
+  backups = var.backups
+  ssh_keys = [var.sam_ssh_key_fingerprint, var.guillaume_ssh_key_fingerprint, var.hubert_ssh_key_fingerprint, var.david_ssh_key_fingerprint]
```

Comment:
Can we get this from an optional configuration file?

Reply:
The thing about this is that the `terraform.tfvars` file has the configurations. So from the gitignore content, any user should know there needs to be a `.tfvars` file with the correct values. However, I'd add a note to the docs to let users know the details.